nemo_curator/modules/fuzzy_dedup.py

```diff
@@ -1438,6 +1436,7 @@ def cc_workflow(self, output_path):
         cc_path = self._run_connected_components(
             deduped_encoded_jaccard_path, deduped_parsed_id_path, output_path
         )
+        self._logger.info(f"End to End time in cc_workflow = {time.time() - st}s")
```

Comment:
Can this also be in the same `Time taken for end to end cc_workflow was 12.345s`

Reply:
Good call out will change